.infrastructure/helm-chart/todoapp/templates/configMap.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{.Chart.Name}}-config
+  namespace: {{.Values.todoapp.namespace}}
```

Comment:
Ensure that `{{.Values.todoapp.namespace}}` is defined in your `values.yaml` file for the `todoapp`. If it's not defined, it could lead to deployment errors.